taichi/ir/statements.h

```diff
@@ -225,6 +256,9 @@ class ExternalPtrStmt : public Stmt {
   TI_DEFINE_ACCEPT_AND_CLONE
 };
 
+/**
+ * A global pointer.
```

Comment:
A bit more explanation on this (on a new paragraph)? For example, this will not show up in the final lowered IR.